modules/convert/user.go

```diff
@@ -50,6 +50,7 @@ func toUser(user *user_model.User, signed, authed bool) *api.User {
 		ID:          user.ID,
 		UserName:    user.Name,
 		FullName:    user.FullName,
+		LoginName:   user.LoginName,
```

Comment:
should we expose this info for all users?!?

I think only admins and the user itselve should have access to this

Reply:
@6543 I've made additional commit. Could you review and see if it makes sense?